Certification-Standard/introduction.md

```diff
@@ -0,0 +1,239 @@
+
+## 2 General model
+
+### 2.1 Scope
+
+The CCS wishes to establish the general concepts and principles of DApp security evaluation and to specify the general model of evauation. This section provides an overview of all the parts of the CCS, it defines all the different terms, abbreviations and concepts used through the rest of the document. It establishes the core principles of the standard:
+
+- Protection Profiles
+- Security Targets
+- Target of Evaluation
+- Evaluation Levels
+
+### 2.2 Terms and Definitions
+
+*RSoulat* Note: Taken from the CC definitions
+*RSoulat* Maybe we could drop the Protection Profiles from a first version and only deal with Security Target
+
+**Adverse action**: An action performed by a Theat agent on an Asset
+
+**Asset**: Entity that the owner of the Target of Evaluation places value on.
+*RSoulat* Example of assets we could consider: Ada, Tokens, NFTs, Wallet private keys
+
+**Attack potential**: Measure of the effort needed to exploit a vulnerability in a Targer of Evaluation (TOE)
+
+**Attack Surface**: Set of logical interfaces to a target, consisting of points through which access to the target's assets or control.
+*RSoulat* I removed "physical" and changed a bit the definition from the CC definition to better fit (I believe) what we're doing.
+
+**Decentralized Application**: Set of software running both on the blockchain, smart contracts, and on classical servers, off-chain code.
+
+**Developer**: Organization responsible for the development of the Target of Evaluation
+
+**Entity**: Identifiable item that is described by a set of properties. For example: subjects, users, objects, information sessions, resources
+*RSoulat* We could include Ada, Tokens, NFTs, Wallets, etc in the Entity definition?
+
+**Evaluation Level**: Well formed package of security assurance requirements representing a point on the predefined assurance scale.
+
+**Evaluation authority**: Body operating an evaluation scheme
+*RSoulat* Could be Auditors, or any "certificate provider", the difference with the CC is that the evaluation authority is **not** the certificate issuer
+
+**Exploitable vulnerability**: Weakness in the Target of Evaluation (TOE) that can be used to violate the Security Functional Requirement (SFRs) in the Operation Environment (OE) for the TOE
+
+**Object**: Entity in the Target of Evaluation that contains or receives information and upon which subjects perform operations.
+*RSoulat* Maybe we could also augment this definition to include UTxOs? 
+
+**Operation Environment**: Enfironment in which the Target of Evaluation (TOE) is operated, consisting of everything that is outside the TOE boundary
+*RSoulat* For example, the ledger, the other contracts, the off-chain code if the TOE is only the on-chain component
```

Comment:
… but it some cases the TOE will be wider, including off-chain code and services.

Reply:
Absolutely, I was just giving some examples. Third-party services will probably always fall under OE though.